domains/2222na.json

```diff
@@ -0,0 +1,9 @@
+{
+  "owner": {
+    "username": "2222Na",
+    "email": "t62648270@gmail.com"
+  },
+  "record": {
+    "CNAME": "89.110.65.214:6999"
```

Comment:
CNAME must be a domain name, if you want to use IPv4, use A record (btw both A and CNAME can not contain ports)
also I can not reach your website, "89.110.65.214 refused to connect."